sync/class.jetpack-sync-module-updates.php

```diff
@@ -5,6 +5,11 @@ class Jetpack_Sync_Module_Updates extends Jetpack_Sync_Module {
 	const UPDATES_CHECKSUM_OPTION_NAME = 'jetpack_updates_sync_checksum';
 
 	private $old_wp_version = null;
+	private $updates = array();
+
+	public function set_defaults() {
```

Comment:
do we need this since we are already doing `private $updates = array();` ?

Reply:
It is there to make the tests pass.